src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2019,6 +2019,7 @@ gregtech.machine.fermenter.hv.name=Advanced Fermenter II
 gregtech.machine.fermenter.ev.name=Advanced Fermenter III
 
 gregtech.machine.fluid_canner.tooltip=Puts fluids into and out of fluid containers
+gregtech.machine.fluid_canner.jei_description=Don't you know that you could use the Fluid Canner Machine to fill Buckets and Fluid Cells?
```

Comment:
I think this should be more dry not personal info (I am not sure how to describe this). Also it should inform about possibility to fill and **empty** any **fluid containers** and then maybe mention Buckets and Fluid Cells as example.

@pyure, @ALongStringOfNumbers What you think regarding this?

Reply:
@LAGIdiot, @pyure, @ALongStringOfNumbers  I've updated the PR with a proposal which takes @LAGIdiot 's suggestions into account... tell me how you feel about it.

Reply:
Agreed with LAGIdiot (mostly because of consistency rather than the tone itself).  Revised version looks good.

Reply:
Revise version accepted